concepticondata/conceptlists/Urban-2016-160.tsv

```diff
@@ -0,0 +1,161 @@
+Urban-2016-160-49	49	river bed	Nature-related and topological concepts	993	RIVER BED
+Urban-2016-160-50	50	root	Nature-related and topological concepts	670	ROOT
+Urban-2016-160-51	51	seed	Nature-related and topological concepts	714	SEED
+Urban-2016-160-52	52	shadow	Nature-related and topological concepts	2275	SHADOW
+Urban-2016-160-53	53	sky	Nature-related and topological concepts	1732	SKY
+Urban-2016-160-54	54	smoke	Nature-related and topological concepts	778	SMOKE (EXHAUST)
+Urban-2016-160-55	55	soil	Nature-related and topological concepts	1228	EARTH (SOIL)
+Urban-2016-160-56	56	spark	Nature-related and topological concepts	518	SPARK
+Urban-2016-160-57	57	spring/well	Nature-related and topological concepts	2144	SPRING OR WELL
+Urban-2016-160-58	58	star	Nature-related and topological concepts	1430	STAR
+Urban-2016-160-59	59	steam	Nature-related and topological concepts	1193	STEAM
+Urban-2016-160-60	60	straw	Nature-related and topological concepts	2299	STRAW
+Urban-2016-160-61	61	sun	Nature-related and topological concepts	1343	SUN
+Urban-2016-160-62	62	swamp	Nature-related and topological concepts	1145	SWAMP
+Urban-2016-160-63	63	tail	Nature-related and topological concepts	1220	TAIL
+Urban-2016-160-64	64	thorn	Nature-related and topological concepts	124	THORN
+Urban-2016-160-65	65	thunder	Nature-related and topological concepts	1150	THUNDER
+Urban-2016-160-66	66	tree	Nature-related and topological concepts	906	TREE
+Urban-2016-160-67	67	valley	Nature-related and topological concepts	921	VALLEY
+Urban-2016-160-68	68	volcano	Nature-related and topological concepts	2291	VOLCANO
+Urban-2016-160-69	69	waterfall	Nature-related and topological concepts	943	WATERFALL
+Urban-2016-160-70	70	wave	Nature-related and topological concepts	978	WAVE
+Urban-2016-160-71	71	wax	Nature-related and topological concepts	1931	BEESWAX
+Urban-2016-160-72	72	whirlpool	Nature-related and topological concepts	1982	WHIRLPOOL
+Urban-2016-160-73	73	airplane	Artifacts	1597	AIRPLANE
+Urban-2016-160-74	74	ball	Artifacts	1845	BALL
+Urban-2016-160-75	75	bed	Artifacts	1663	BED
+Urban-2016-160-76	76	belt	Artifacts	772	BELT
+Urban-2016-160-77	77	boat	Artifacts	1844	BOAT
+Urban-2016-160-78	78	car	Artifacts	1111	CAR
+Urban-2016-160-79	79	chair	Artifacts	1476	CHAIR
+Urban-2016-160-80	80	clock	Artifacts	1323	CLOCK
+Urban-2016-160-81	81	glasses	Artifacts	2292	GLASSES
+Urban-2016-160-82	82	house	Artifacts	1252	HOUSE
+Urban-2016-160-83	83	key	Artifacts	1472	KEY
+Urban-2016-160-84	84	knife	Artifacts	1352	KNIFE
+Urban-2016-160-85	85	ladder	Artifacts	1464	LADDER
+Urban-2016-160-86	86	mirror	Artifacts	1171	MIRROR
+Urban-2016-160-87	87	needle	Artifacts	1382	NEEDLE (FOR SEWING)
+Urban-2016-160-88	88	paper	Artifacts	648	PAPER
+Urban-2016-160-89	89	pen	Artifacts	794	PEN
+Urban-2016-160-90	90	rope	Artifacts	1218	ROPE
+Urban-2016-160-91	91	scissors	Artifacts	1328	SCISSORS
+Urban-2016-160-92	92	shoe	Artifacts	1381	SHOE
+Urban-2016-160-93	93	road/street/way	Artifacts	2457	PATH OR ROAD
```

Comment:
Urban-2011-160-93 was mapped to 667 ROAD

Reply:
Yes, this is a rather ambiguous case. I would probably check with the kind of languages (most importantly, cultures where these languages are spoken) when making a decision on this one.